code/game/machinery/_machinery.dm

```diff
@@ -501,7 +501,7 @@
 
 
 /obj/machinery/can_interact(mob/user)
-	if((machine_stat & (NOPOWER|BROKEN)) && !(interaction_flags_machine & INTERACT_MACHINE_OFFLINE)) // Check if the machine is broken, and if we can still interact with it if so
+	if((machine_stat & (NOPOWER|BROKEN) & !istype(/obj/machinery, /obj/machinery/plumbing)) && !(interaction_flags_machine & INTERACT_MACHINE_OFFLINE)) // Check if the machine is broken, and if we can still interact with it if so
```

Comment:
This istype will always return false because you're passing in /obj/machinery instead of src.
That & in ```BROKEN) & !istype``` should be an &&. & compares individual bits and since NOPOWER is 2 and istype would return either 1 or 0 this if statement will never be true unless the machine is broken.

Regardless, you shouldn't even need to touch this proc, you could just do this by giving plumbing machines the INTERACT_MACHINE_OFFLINE tag. You'd just have to make sure to also give it to the ones that don't actually inherit from obj/machinery/plumbing/. I know at least IV drips don't.

Reply:
that is... a very good and helpful comment and a much better solution, thank you!
I added the flag to all plumbing machinery, plus IV_drips and the limb grower